fract4dgui/fourway.py

```diff
@@ -19,18 +19,16 @@ def __init__(self, text, tip):
         self.last_x = 0
         self.last_y = 0
         self.text = text
-        Gtk.DrawingArea.__init__(self, tooltip_text=tip)
-
-        self.set_size_request(53, 53)
+        Gtk.DrawingArea.__init__(self, tooltip_text=tip, width_request=53, height_request=53)
 
         self.set_events(
-            Gdk.EventMask.BUTTON_RELEASE_MASK |
-            Gdk.EventMask.BUTTON1_MOTION_MASK |
-            Gdk.EventMask.POINTER_MOTION_HINT_MASK |
-            Gdk.EventMask.ENTER_NOTIFY_MASK |
-            Gdk.EventMask.LEAVE_NOTIFY_MASK |
-            Gdk.EventMask.BUTTON_PRESS_MASK |
-            Gdk.EventMask.EXPOSURE_MASK
+            Gdk.EventMask.BUTTON_RELEASE_MASK
```

Comment:
Same comment as before about the consistency of this with the rest of the code base.